core/modules/util/ThreadPool.cc

```diff
@@ -204,6 +220,21 @@ bool ThreadPool::release(PoolEventThread *thrd) {
 /// Change the size of the thread pool.
 void ThreadPool::resize(unsigned int targetThrdCount) {
     {
+        LOGS(_log, LOG_LVL_WARN, "&&&WARN ThreadPool::resize " << targetThrdCount);
```

Comment:
Since `resize` is a normal  and probably rather frequent operation then reporting it at level `WARN` is may be a bit of an "over-kill" for the production code. Perhaps `DEBUG` or even `TRACE` would still work?